### DISCUSSION TITLE
Add ability to add custom live-sync delay to windows other than the main stream

### DISCUSSION THREAD

**Discussion by jameslemer** [f1multiviewer/issue-tracker#575] [Feature requests]
Posting here after talking to the team through [a ticket](https://canary.discord.com/channels/982375740764983366/1244249814565388312) "12-live-syncing-tips-i-notice-th"

When using multiviewer, I always notice that after clocking the sync button, the main stream is consistently 2 seconds behind the other windows. 
I will have 30 as the target latency and then when I click the sync button, the streams will go to around 41 latency. But then I have to manually adjust the sync +2 seconds on all of the other streams for it to sync properly. This is the same every single race for me and after checking the system clock etc.

My request would be for a feature to change the delay of the sync button within the settings. So that if you click the sync button, it will sync, but to your custom delay so that it syncs properly on clicking it. 

As I discussed with the team, I could just -2 seconds from the main stream so that it syncs with all of the other streams. however, I watch live with friends, so that would put me out of sync with my friends. And it would be nice to have it work straight away every time.

You could also maybe save the sync timing to your layout when you save a layout? 

Hopefully, that makes sense. Reach out if you have questions

mod edit: add link to ticket

**Comment by LapsTimeOFF** [comment]
> You could also maybe save the sync timing to your layout when you save a layout?

The problem is that this little delta can be inconsistant from sessions to sessions.
We are always doing our best to make the sync system better each day, as a reminder you can check our [docs](https://multiviewer.app/docs/usage/syncing-streams) on how to make your sync better. Feel free to also raise up any syncs issues in #syncs on our discord server.
